apiserver/facades/client/machinemanager/state.go

```diff
@@ -96,6 +98,11 @@ type Unit interface {
 	UnitTag() names.UnitTag
 }
 
+func (m machineShim) VerifyUnitsSeries(unitNames []string, series string, force bool) error {
+	_, err := m.Machine.VerifyUnitsSeries(unitNames, series, force)
```

Comment:
Given the underlying function has a return value, why is it that we don't need to do anything with that value? Would it be a list of things we should recurse on?

Reply:
Normally it'd return a slice of unit tags, both principals and subordinates.  It depend on how the next pieces are implemented, where it's useful at this point.  Posed question in google doc